server/routes/oauth/discord.ts

```diff
@@ -51,6 +51,25 @@ export default defineEventHandler(async event => {
 
   session.data.discordId = user.id
 
+  // makes sure the user is in the guild
+  if (!session.data.guildMemberAdded) {
+    console.log('adding user to guild', session.data.guildMemberAdded)
+    await $fetch(
+      `https://discord.com/api/guilds/${config.discord.guildId}/members/${session.data.discordId}`,
+      {
+        method: 'PUT',
+        body: JSON.stringify({
+          access_token,
+        }),
```

Comment:
nit: no stringify needed when using `$fetch` 😋

```suggestion
        body: {
          access_token,
        },
```

Reply:
aw yeah I forgot how great is $fetch. thanks